README.md

```diff
@@ -1,33 +1,41 @@
 # Wayland McWayface (JVM-edition)
 
-Repository contains implementation of Wayland McWayface [Part 3](https://drewdevault.com/2018/02/28/Writing-a-wayland-compositor-part-3.html) by Drew DeVault. Jextract utility provided by project panama and `foreign` api are used directly for including bindings of wlroots and wayland within java. The `foreign` api can be seen in `usr.include.stdlib` in the scala sources.
+Repository contains implementation of Wayland McWayface [Part 3](https://drewdevault.com/2018/02/28/Writing-a-wayland-compositor-part-3.html) by Drew DeVault. 
+Jextract utility provided by project panama and `foreign` api are used directly for including bindings of wlroots and wayland within java. 
+The `foreign` api can be seen in `usr.include.stdlib` in the scala sources.
 
 ## Dependencies
 
-[build ea+44 of project panama on jdk 13](https://jdk.java.net/panama/) is required to build and run the project. The wayland development libraries, the wlroots library and development libraries, clang, libpixman's development libraries, libxkbcommon are also needed for the project to build and run. Additional libraries maybe required.
+[build ea+44 of project panama on jdk 13](https://jdk.java.net/panama/) is required to build and run the project. 
+The wayland development libraries, the wlroots library and development libraries, clang, libpixman's development libraries, libxkbcommon are also needed for the project to build and run. 
+Additional libraries maybe required.
 
 ## Building
 
 Run `sbt compile` in the root of the project directory to build the project.
 
 Check the following settings in build.sbt if build fails:
 
+This is an implementation of Wayland McWayface [Part 3](https://drewdevault.com/2018/02/28/Writing-a-wayland-compositor-part-3.html) by Drew DeVault. 
+In order to have bindings of wlroots and wayland within java, I've made use of the jextract utility provided by project panama. 
+I've also made use of the `foreign` api directly which can be seen in `usr.include.stdlib` in the scala sources.
 ```scala
 xdgShellProtocolLocation := file("/usr/share/wayland-protocols/unstable/xdg-shell/xdg-shell-unstable-v6.xml")
 
 includeDirectory := file("/usr/include")
 
 libraryDirectory := file("/usr/lib64")
 ```
-Settings in build.sbt should point to xdg-shell-unstable-v6.xml, include path, and the path that contains the .so files on your system.
```

Comment:
Please undo this change. I'm trying to indicate that the settings **should** point, but may not.